README.md

```diff
@@ -88,6 +89,8 @@ Never commit directly to the `production` branch. Commit your changes to a devel
 
 Once merged, you will need to connect to our production Airflow host on the COA network, then pull down your changes from Github. Airflow will automatically load any DAG changes within five minutes. Activate your DAG through the Airflow web interface at `https://airflow.austinmobility.io/`.
 
+#### If there has been a change to the code in the repo:
```

Comment:
After I wrote this, I realized there could be occasion where both a GitHub and Docker pull are needed. Maybe I am overthinking it? 🤷

Reply:
Naw, you're totally on the right track. My take, and check my math for sure, is that a `git pull` is good in all updates and a `docker compose pull` should be done, in addition, when the image has been updated.

Reply:
awesome - i added a `git pull` to the second set of steps too so it is done either way. 🙏